python-package/xgboost/dask.py

```diff
@@ -1522,14 +1486,16 @@ def fit(
         sample_weight: Optional[_DaskCollection] = None,
         base_margin: Optional[_DaskCollection] = None,
         eval_set: Optional[List[Tuple[_DaskCollection, _DaskCollection]]] = None,
-        eval_metric: Optional[Union[str, List[str], Callable]] = None,
-        sample_weight_eval_set: Optional[List[_DaskCollection]] = None,
-        early_stopping_rounds: int = None,
+        eval_metric: Optional[Callable] = None,
+        early_stopping_rounds: Optional[int] = None,
         verbose: bool = True,
-        feature_weights: _DaskCollection = None,
+        xgb_model: Optional[Booster] = None,
+        sample_weight_eval_set: Optional[List[_DaskCollection]] = None,
+        feature_weights: Optional[_DaskCollection] = None,
         callbacks: Optional[List[TrainingCallback]] = None
     ) -> "DaskXGBClassifier":
         _assert_dask_support()
+        assert xgb_model is None, '`xgb_model` is not supported, load the model instead.'
```

Comment:
Why is `xgb_model` not supported for DaskXGBRanker?

Reply:
Written before making the decision of flushing model.  Will correct that.

Reply:
Done.